Examples/VaporExample/README.md

```diff
@@ -8,22 +8,22 @@ This application is intended to demonstrate best practices for integrating the d
 
 The application contains both a REST API as well as a minimal frontend built with Vapor's templating language [Leaf](https://github.com/vapor/leaf).
 
-This application require Swift 5.2 and MongoDB 3.6+. It will run on Linux as well as macOS 10.15+.
+This application requires Swift 5.5+ and MongoDB 3.6+. It will run on Linux as well as macOS 12+.
+
+If you are looking for an example application that uses older Swift versions prior to the introduction of structured concurrency, please refer to a previous version of this application and README [here](https://github.com/mongodb/mongo-swift-driver/tree/5d9fad121d7cb3ded61087de300ff007766ccd55/Examples/VaporExample).
```

Comment:
I considered trying to maintain two versions of the example but I don't know if it is worth the effort, so just linking to the latest commit seemed like a reasonable way to not totally lose the info but strongly push users toward async/await.

Reply:
I think a back link strikes a good compromise, no need to maintain a separate example here for the reasons you list IMO.

Reply:
I think linking back to the previous version is sufficient for now, as it keeps the ELG examples accessible while making async/await the more discoverable option. If there's significant interest in the ELG examples or we need to make changes to them for some reason we still have the opportunity to reintroduce them down the line, but for now I don't think it's super necessary.